src/main/java/com/softserve/messaging/model/MessageHandler.java

```diff
@@ -1,34 +1,32 @@
-package com.softserve.message_sender.domain;
+package com.softserve.messaging.model;
 
+import lombok.AllArgsConstructor;
 import lombok.Data;
 import lombok.NoArgsConstructor;
 
 import javax.persistence.Entity;
 import javax.persistence.GeneratedValue;
 import javax.persistence.GenerationType;
 import javax.persistence.Id;
+import java.time.ZonedDateTime;
+import java.util.UUID;
 
 @Entity
 @Data
 @NoArgsConstructor
-public class UserReceipt {
+@AllArgsConstructor
+public class MessageHandler {
```

Comment:
what does this entity handle?

Reply:
wrong naming, splitted this class into  userReceipt and userValidation